CI/macos/02_build_obs.sh

```diff
@@ -69,6 +69,12 @@ _configure_obs() {
         UNITTEST_OPTIONS="-DENABLE_UNIT_TESTS=ON"
     fi
 
+    if [ "${CI}" -a "${ARCH}" = "x86_64" ]; then
```

Comment:
I think this is a great work and tried to communicate with my WHIP server. And I found when building on my x86 macOS machine I had to remove CI checking on this line to specify the architecture.
It seems to work without the CI environment check on this line, so it would be helpful when I try something on macOS x86 machines if it could be modified like below.

```diff
--- a/CI/macos/02_build_obs.sh
+++ b/CI/macos/02_build_obs.sh
@@ -69,7 +69,7 @@ _configure_obs() {
         UNITTEST_OPTIONS="-DENABLE_UNIT_TESTS=ON"
     fi

-    if [ "${CI}" -a "${ARCH}" = "x86_64" ]; then
+    if [ "${ARCH}" = "x86_64" ]; then
       Rust_CARGO_TARGET="x86_64-apple-darwin"
     else
       Rust_CARGO_TARGET="aarch64-apple-darwin"
```